client/my-sites/plan-features/item.jsx

```diff
@@ -27,15 +28,25 @@ export default function PlanFeaturesItem( {
 		onMouseLeave( event.currentTarget, description );
 	};
 
+	const mouseEvents = {
+		onMouseEnter: handleOnMouseEvent,
+		onMouseLeave: handleOnMouseLeave
+	};
+	const hoverOnRow = (
+		abtest( 'plansDescriptions' ) === 'ascendingPriceEagerDescription' ||
+		abtest( 'plansDescriptions' ) === 'descendingPriceEagerDescription'
+	);
+
 	return (
-		<div className="plan-features__item">
+		<div className="plan-features__item"
+			{ ...( hoverOnRow && mouseEvents ) }
```

Comment:
Yes, I noticed and tried to changed that and decided its actually better.
> One caveat of interaction of new refactored popover and my changes is that on right column the popover displays right of row instead of right-of-icon. I actually like this behavior. I also created a commit that gets rid of that and I think its much worse:

- commit 9f73f66
- videeo: https://www.youtube.com/watch?v=1H5Noe91Ho0&feature=youtu.be